src/include/splash/core/DCDataSet.hpp

```diff
@@ -34,6 +34,8 @@
 #include "splash/CollectionType.hpp"
 #include "splash/basetypes/ColTypeDim.hpp"
 
+herr_t H5Ewalk_strings(unsigned n, const H5E_error2_t *err_desc, void *client_data);
```

Comment:
I agree with @ax3l , this should be more common, both the declaration and the implementation.